core/src/test/java/org/apache/iceberg/hadoop/TestStaticTable.java

```diff
@@ -60,6 +60,19 @@ public void testCannotBeDeletedFrom() {
         () -> staticTable.newDelete().deleteFile(FILE_A).commit());
   }
 
+  @Test
+  public void testCannotDoIncrementalScanOnMetadataTable() {
+    table.newAppend().appendFile(FILE_A).commit();
+
+    for (MetadataTableType type : MetadataTableType.values()) {
+      Table staticTable = getStaticTable(type);
+      AssertHelpers.assertThrows("Static tables do not currently support incremental scans",
```

Comment:
I think that a better place for this test is now `TestMetadataTableScans` since it supports all metadata tables. Also, the context message still says "Static tables . . .".